release.yaml

```diff
@@ -1,3 +1,5 @@
+prometheus-federator:
+- 1.0.0+up0.1.1
```

Comment:
we're still using 0.1.1. Described in https://github.com/rancher/rancher/issues/38979#issuecomment-1251628530, Project Monitoring V2 versioning will be independent of Rancher versioning since there is no co-dependency at a binary level and a chart level change (like modifying the Rancher version above) can be patched.